server/scripts/start-server.sh

```diff
@@ -1,9 +1,3 @@
 #!/bin/bash
 
-# Note this will fail from in a Windows environment due to resolving failures for .exe files. Use WSL context instead
-if [ "$1" == "--sync" ]; then
```

Comment:
Isn't this still useful for syncing dependencies without doing the setup again?

Reply:
thats a good point, reverted this change and modiefied Dockerfile instead